plugins/plugin-languageserver/che-plugin-languageserver-ide/src/main/java/org/eclipse/che/plugin/languageserver/ide/quickopen/EditorQuickOpenEntry.java

```diff
@@ -38,7 +38,7 @@ protected TextRange getTextRange() {
   public boolean run(Mode mode) {
     if (mode == Mode.OPEN) {
       String filePath = getFilePath();
-      editorHelper.openPath(filePath, getTextRange());
+      editorHelper.openLocation(filePath, getTextRange());
```

Comment:
I would just scrap the EditorQuickOpenEntry (it is only extended once) and instead make a QuickOpenEntry derived class that holds a SymbolInformation object directly. Then just use the location to open the editor.